docs/modules/ROOT/pages/vshn-managed/mariadb/maintenance.adoc

```diff
@@ -0,0 +1,24 @@
+= Scheduled Maintenance
+
+By default, there will be a security maintenance on Tuesday night (Central Europe Time).
+
+NOTE: For more information about maintenance procedures, please visit https://products.docs.vshn.ch/products/appcat/redis.html#_recurring_maintenance[the product description].
+
+The day and time can be adjusted as described below. Times are in the CET time zone.
+
+.Redis instance with custom maintenance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: Redis
+metadata:
+  name: redis-app1-prod
```

Comment:
file is named mariadb/maintenance but contains Redis values

Reply:
Fixed